### PR TITLE
add button for restore default ratio

### PR DESCRIPTION
**Pull request by myarjunar**
inasafe 4 0 0b0 options_017

### CODE REVIEW

**Comment by Gustry** [comment]
I would add the button in the toolbar in the bottom, no ?

**Comment by myarjunar** [comment]
So, my consideration of putting the button there is because it will only restore the global default tab. That's what I'm getting from the issue. Am I right? Or it should restore the default value for the whole setting? If so, you're right, we're gonna put it at the bottom.

**Comment by Gustry** [comment]
Ah yes correct, thanks !

File: safe/gui/tools/options_dialog.py [hunk 1]
```diff
@@ -545,3 +557,14 @@ def save_default_values(self):
                 parameter.guid,
                 parameter.value
             )
+
+    def restore_defaults_ratio(self):
+        '''Restore InaSAFE default ratio.'''
```
Comment (ismailsunni) [inline safe/gui/tools/options_dialog.py]: Triple double-quote

**Comment by Gustry** [comment]
You can have a look to landscape, it's a an easy fix ;-)

**Comment by ismailsunni** [comment]
Hi @myarjunar 
I tried your PR, but it seems it doesn't do what it should do. The button should reset the value when clicked with the default one. You can get the default value from https://github.com/inasafe/inasafe/blob/develop/safe/definitions/default_values.py#L16

**Comment by myarjunar** [comment]
So the previous one will only restore to defaults on the moment the user edit the value then directly click the button, it is not working when the user edit the value, then click ok, then reopen it, then click the the restore defaults button. Hope current commit will fix.

**Comment by ismailsunni** [comment]
Hi @myarjunar 
I checked and it looks good and has proper behaviour. But I saw you recreate the widget in your code. Can you make it only to update the value in the combobox ?

**Comment by timlinux** [comment]
See #3463

**Comment by ismailsunni** [comment]
I think my we can merge this since it has fixed the problem. LGTM. We can visit again later for better implementation.

**Pull request merged**